### PR TITLE
Fix: package.json upgrade

### PR DESCRIPTION
**Pull request by yarpo**
# Description

We have an issue with `yarn audit` that was failing on master branch.

All we had to do was just make sure two packages (tar and fstream) will be upgraded.
https://github.com/sass/node-sass/issues/2676#issuecomment-498078788

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,6 +66,7 @@
     "extract-text-webpack-plugin": "^2.1.0",
     "file-loader": "^0.10.1",
     "formidable": "^1.1.1",
+    "fstream": "^1.0.0",
```
Comment (tchow8) [inline package.json]: I think its probably safer to use 1.0.12 (the fixing version) as the minimum version.
With the ^, it'll automatically retrieve the latest minor version, so this should still work though.

**Comment by dtaylor7** [comment]
seems good but please wait for https://github.com/hmcts/div-petitioner-frontend/pull/566

File: package.json [hunk 2]
```diff
@@ -99,6 +100,7 @@
     "stryker": "^0.23.0",
     "superagent": "3.8.1",
     "superagent-proxy": "^1.0.2",
+    "tar": "^2.0.0",
```
Comment (tchow8) [inline package.json]: Similarly to above, 2.2.2 is the fixing version. May be safer to have that as the minimum.

Comment (yarpo) [reply package.json]: weird thing is that when I tried 2.2.2 it wasn't working :/

Comment (yarpo) [reply package.json]: hm... works now :)

Comment (yarpo) [reply package.json]: ^2.2.2

Comment (tchow8) [reply package.json]: Hmmm strange, its available on the package manager, https://www.npmjs.com/package/tar/v/2.2.2 so thats weird.

**Pull request merged**